modules/Template/Data.yaml

```diff
@@ -298,6 +297,20 @@ classes:
       requiresComponent: ''
       required: false
 
+  ChIPSeqTemplate:
+    is_a: EpigenomicsAssayTemplate
+    description: Template for describing raw data from ChIP-Seq assays, typically used to identify DNA-protein interactions.
+    slots:
+      - antibodyID
```

Comment:
You can add a more specific `ChIPseqAntibodyName` property with the NCIT meaning as suggested and use it here. We can either keep it free text or import a list of ChIPseq antibody names, but I'd prefer people use the `antibodyID` as the preferred field anyway -- you can make `antibodyID` required. For additional context, it looks like HTAN just uses single free text `antibodyName` primarily https://github.com/ncihtan/data-models/blob/4b9bb3cfcbb979694479a4a60b7dac71becfbf96/HTAN.model.csv#L226

Reply:
Is the antibodyID already in free text form?
https://github.com/nf-osi/nf-metadata-dictionary/blob/main/modules/props.yaml#L73-L75

Reply:
Yes

Reply:
If we make this `antibodyID` slot required, would this also effect other templates that might been using the slot?

Reply:
Yes, because the feature to set contextual required is not implemented in schematic yet. In this case, I think it's OK to make it required across all templates that use it -- you would only add antibodyID to the template if you really needed it anyway.